astro/src/content/docs/apis/_tenant-request-body.mdx

```diff
@@ -441,6 +449,10 @@ import TransactionTypes from 'src/content/docs/apis/_transaction-types.mdx';
     When enabled, all refresh tokens will be revoked when a user enables multi-factor authentication for the first time. This policy will not be applied when adding subsequent multi-factor methods to the user.
   </APIField>
 
+  <APIField name="tenant.jwtConfiguration.refreshTokenRevocationPolicy.onOneTimeTokenReuse" type="Boolean" optional defaults="false" since="1.55.0">
```

Comment:
```suggestion
  <APIField name="tenant.jwtConfiguration.refreshTokenRevocationPolicy.onOneTimeTokenReuse" type="Boolean" optional defaults="false" since="1.55.1">
```